core/src/main/java/greencity/controller/HabitAssignController.java

```diff
@@ -173,6 +173,37 @@ public ResponseEntity<UserShoppingAndCustomShoppingListsDto> getUserShoppingList
                 .getUserShoppingListItemAndUserCustomShoppingList(userVO.getId(), habitId, locale.getLanguage()));
     }
 
+    /**
+     * Method that update UserShoppingList and CustomShopping List.
+     *
+     * @param habitId  {@link HabitVO} id.
+     * @param userVO   {@link UserVO} instance.
+     * @param locale   needed language code.
+     * @param listsDto {@link UserShoppingAndCustomShoppingListsDto} instance.
+     */
+    @ApiOperation(value = "Update user and custom shopping lists",
+        notes = "If item are present in the db, method update it\n"
+            + "If item doesn't present in the db and id is -1, method try to add it to user\n"
```

Comment:
I guess it is not a good idea to use `-1`
It is better to check if `id == null`, and if it is null create a new record in DB